src/server/index.jsx

```diff
@@ -192,6 +198,7 @@ server.get(
     injectDefaultCacheHeader,
     injectReferrerPolicyHeader,
     injectResourceHintsHeader,
+    injectPlatformToRequestChain,
```

Comment:
Could we add Header on the end for consistency `injectPlatformToRequestChainHeader`